main.js

```diff
@@ -0,0 +1,45 @@
+const button = document.getElementById('burger-menu');
+const menu = document.querySelector('#nav_list');
+const menuLinks = [...menu.children];
+const img = button.querySelector('img');
+const body = document.querySelector('body');
+
+button.addEventListener('click', () => {
+  if (img.getAttribute('data-toggled') === 'false') {
+    img.src = 'images/exit-menu.svg';
+    img.setAttribute('data-toggled', 'true');
+  } else {
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you add a `cursor pointer` property on the styling of your **Exit-Menu or CloseMenu** button to help users know that is clickable when it has hovered.